doc/ale-yaml.txt

```diff
@@ -366,5 +366,37 @@ g:ale_yaml_gitlablint_options                     *g:ale_yaml_gitlablint_options
   This variable can be set to pass additional options to gll.
 
 
+===============================================================================
+yq                                                                  *ale-yaml-yq*
+
+Website: https://github.com/mikefarah/yq
+
+
+Installation
+-------------------------------------------------------------------------------
+
+Install yq: >
+
+  wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY}.tar.gz -O - | tar xz && mv ${BINARY} /usr/bin/yq
+
+Options
+-------------------------------------------------------------------------------
+
+g:ale_yaml_yq_executable                               *g:ale_yaml_yq_executable*
+                                                       *b:ale_yaml_yq_executable*
+  Type: |String|
+  Default: `'gll'`
```

Comment:
This does not look right? should this be `yq` instead of `gll`?

Reply:
Copy-paste issue, should be fixed now